source/applications/advanced/create_depth_map.py

```diff
@@ -12,57 +13,90 @@
 from utils.paths import get_sample_data_path
 
 
-def _main():
-
-    app = zivid.Application()
+def _point_cloud_to_cv_z(point_cloud):
+    """Get depth map from frame.
 
-    filename_zdf = Path() / get_sample_data_path() / "Zivid3D.zdf"
-    print(f"Reading {filename_zdf} point cloud")
-    frame = zivid.Frame(filename_zdf)
+    Args:
+        frame: Zivid point cloud
 
-    # Getting the point cloud
-    depth_map = frame.point_cloud().copy_data("z")
+    Returns:
+        OpenCV Depth map
 
+    """
+    depth_map = point_cloud.copy_data("z")
     depth_map_uint8 = (
         (depth_map - np.nanmin(depth_map)) / (np.nanmax(depth_map) - np.nanmin(depth_map)) * 255
     ).astype(np.uint8)
 
-    # Applying color map
-    depth_map_color_map = cv2.applyColorMap(depth_map_uint8, cv2.COLORMAP_JET)
+    depth_map_color_map = cv2.applyColorMap(depth_map_uint8, cv2.COLORMAP_VIRIDIS)
 
     # Setting nans to black
     depth_map_color_map[np.isnan(depth_map)[:, :]] = 0
 
-    rgba = frame.point_cloud().copy_data("rgba")
+    return depth_map_color_map
+
+
+def _point_cloud_to_cv_bgr(point_cloud):
+    """Get bgr image from frame.
+
+    Args:
+        frame: Zivid point cloud
+
+    Returns:
+        OpenCV BGR image
```

Comment:
Same here. You are just returning an ndarray:
![image](https://user-images.githubusercontent.com/50012606/97440983-79eea000-1928-11eb-95ec-ebb5fdf49132.png)

Reply:
done.